conjureup/ui/views/spellpicker.py

```diff
@@ -40,7 +40,7 @@ def _build_widget(self):
         total_items = [
             Padding.center_60(HR())
         ]
-        for spell in self.spells:
+        for spell in sorted(self.spells):
```

Comment:
why sort the spells here? 
As it was, they should just be in the order they were found in the yaml, which we can control. we might want to be able to pick an order other than alphabetical